explorer/ExplorerGrammar.ts

```diff
@@ -27,12 +26,8 @@ export const ExplorerGrammar: Grammar = {
     table: {
         ...UrlCellDef,
         keyword: "table",
-        terminalOptions: OwidDatasets.map((keyword) => ({
-            keyword,
-            description: "",
-            cssClass: "",
-        })),
         valuePlaceholder: "",
+        regex: new RegExp(`(${UrlCellDef.regex?.source ?? ""}|^[\\w -()]+$)`), // URL or OWID dataset name
```

Comment:
Nice! That makes sense, we could probably drop this feature entirely at some point.

Reply:
By feature you mean the ability to refer to OWID datasets directly, right?
Makes sense to me. [The way it works might be quite brittle](https://github.com/owid/owid-grapher/blob/43151675072ecdeba0e2a5cbce637e9035252c37/explorer/ExplorerProgram.ts#L399-L402), anyway.